partner_import_helper/__manifest__.py

```diff
@@ -13,7 +13,10 @@
     'depends': [
         'import_helper_base',
         'phone_validation',  # would be nice to avoid depending on it ?
```

Comment:
```suggestion
    'depends': [
        'account',
        'import_helper_base',
        'l10n_fr',
        'phone_validation',  # would be nice to avoid depending on it ?
```

- Must depend on "l10n_fr" (at least, maybe "l10n_fr_siret") to use the field "siret" (and "siren", etc.)
- Must depend on "account" to use the "default" payment terms